lib/editor/util/ui.js

```diff
@@ -85,5 +85,14 @@ export const GTFS_ICONS = [
     title: 'Edit Booking Rules',
     label: 'Booking Rules',
     flexOnly: true
+  },
+  {
+    id: 'locationgroup',
```

Comment:
Sort attributes

Reply:
I think the order of fields in all of these objects was deliberately chosen to make the large list more readable.